ember/app/templates/restaurant.hbs

```diff
@@ -75,6 +75,12 @@
               {{model.restaurant.description}}
             </p>
           </div>
+          <div class="restaurant-card">
+            <h2 class="restaurant-card-title">Where to find {{model.restaurant.name}}</h2>
+            <p class="restaurant-card-bodytext">
+              {{google-map markerLat=model.restaurant.latitude markerLng=model.restaurant.longitude}}
+            </p>
+          </div>
```

Comment:
This code should not be here since it is not related to this PR/Feature.

Just take care when it comes to this. Having the same code on multiple branches can make your life tough when you merge one of the PRs because conflicts might arise in other PRs that have the same code in it.